### ISSUE TITLE
de: missing de::from_* methods?

### ISSUE THREAD

**Issue by lucab** [arcnmx/serde-ini#1]
I was looking at this crate in order to deserialize some INI files directly into their Rust struct with the usual serde idiom:
```
let data: MyStruct = serde_ini::from_str(input)?;
```

However I don't see any of the usual `de::from_*()` methods. Are they still pending or am I missing something?

I saw you have a public `Parser` with an iterator interface and several `From` implementations, but I'm looking for something with direct deserialization and access to sections/values instead.

**Comment by arcnmx** [comment]
Yeah, they're probably missing and should be added. Those are just convenience methods though, you can still create and use the deserializer like:

```rust
let data = MyStruct::deserialize(&mut ini::Deserializer::new(ini::Parser::from_str(input)));
```

**Comment by arcnmx** [comment]
Fixed by #2
